### PR TITLE
user validations and associations

### PR DESCRIPTION
**Pull request by Tiago-O**
please review

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,13 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :repos
+  has_many :docs, through: :repos
+  has_many :versions
+  has_many :comments
+
+  validates :first_name, presence: true, length: { minimum: 2 }
```
Comment (diogobroco) [inline app/models/user.rb:12]: I would change to min 3

**Review by diogobroco — COMMENTED** [review]
check mycomment

File: app/models/user.rb (hunk 1 above)
Comment (Tiago-O) [reply app/models/user.rb:12]: there are name with 2 letters ;)

Comment (diogobroco) [reply app/models/user.rb:12]: OK

**Pull request merged**